doc/source/dev/missing-bits.rst

```diff
@@ -0,0 +1,260 @@
+
+.. [*] This function accepts positional arguments only, but the docstring
+       documents the argument as `lambda`.
+
+
+Implicit string continuation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+When an implicit continuation is used to create a long literal string, the
+string is generally split at a space, and the space is included in the
+second part of the string.
+
+  Yes::
+
+      long_string = ("This is a very long string that demonstrates"
+                     " this guideline.")
+
+  No::
+
+      long_string = ("This is a very long string that demonstrates "
+                     "this guideline.")
+
+
+Documentation Guidelines
+------------------------
+
+Use of LaTeX
+~~~~~~~~~~~~
+[TBD.  In the past, we had an unwritten rule that LaTex markup should be
+restricted to the Notes section.  We don't follow that rule these days.
+Should we?]
+
+
+Use "must", not "should"
+~~~~~~~~~~~~~~~~~~~~~~~~
+When specifying a required condition on the input parameters, the
+word "must" is preferable to "should".  For many English speakers,
+"must" implies a stronger constraint than "should",  e.g. "I must
+have oxygen to live" versus "I should exercise more".
+
+    Yes::
+
+            Parameters
+            ----------
+            x : float
+                x must be nonnegative.
+
+    No::
+
+            Parameters
+            ----------
+            x : float
+                x should be nonnegative.
+
+
+Use of the 'versionadded' markup
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+* For a new function, the 'versionadded' markup goes in the "Notes" section,
+  *not* in the description at the beginning of the docstring.
+* For a new argument added to an existing function,  two locations have been
+  used for the the 'versionadded' markup, [TBD: which is preferred?]:
+
+  * At the end of the description of the argument in the "Parameters" section
+  * In the "Notes" section.  In this case, the `versionadded` markup
+    wouldn't be used.  Instead, the new addition is noted with a plain
+    text comment.
+
+
+Citing wikipedia articles in the "References" section
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+It is acceptable to use wikipedia articles as references.
+When creating the citation for the reference, include the article title,
+the name "Wikipedia" (similar to how one gives a journal title), and the
+URL.
+
+    Yes::
+
+        .. [1] "Zeta Distribution", Wikipedia,
+               https://en.wikipedia.org/wiki/Zeta_distribution
+
+    No::
+
+        .. [1] https://en.wikipedia.org/wiki/Zeta_distribution    
+
+
+Use of ``np`` in the "Examples" section
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Do not include ``import numpy as np`` in the code given in the "Examples"
+section.  The NumPy Docstring standard says "The examples may assume that
+``import numpy as np`` is executed before the example code in numpy."  That
+statement makes the import *optional*; this guideline says explicitly
+that the import statement must not be included.
+
```

Comment:
Speaking of examples, I would add the new way of not using seeds. I can write a suggestion here if you want?

Reply:
Yes, that would be great!

Reply:
Done bellow, hence resolving this one.